packages/core/src/commands/command.ts

```diff
@@ -284,19 +284,26 @@ export abstract class BaseCommand extends Command {
         return this.getNetworks().map(network => ({ title: network, value: network }));
     }
 
-    protected async restartProcess(processName: string) {
+    protected async restartProcess(processName: string, showPrompt: boolean = true) {
         if (processManager.isRunning(processName)) {
-            await confirm(`Would you like to restart the ${processName} process?`, () => {
-                try {
-                    cli.action.start(`Restarting ${processName}`);
-
-                    processManager.restart(processName);
-                } catch (error) {
-                    this.error(error.message);
-                } finally {
-                    cli.action.stop();
-                }
-            });
+            if (showPrompt) {
+                await confirm(`Would you like to restart the ${processName} process?`, () => {
+                    this.restartRunningProcess(processName);
+                });
+            } else {
+                this.restartRunningProcess(processName);    
+            }
+        }
+    }
+
+    protected restartRunningProcess(processName: string) {
```

Comment:
The method name implies that the `if (processManager.isRunning(processName)) {` check is performed in it.

Reply:
Shall I switch out the names of the `restartProcess` and `restartRunningProcess` methods?